### PR TITLE
Italian translation updates

### PR DESCRIPTION
**Pull request by erciccione**
-Edited typo
-deleted obsolete lines
-changed many unfinished in finished
-updated missing lines

It's my fist pull request, let me know if i did anything stupid please

### CODE REVIEW

**Comment by Jaqueeee** [comment]
Looks good. But you should rebase and squash those to commits into 1.

`rebase -i origin/master`
replace "pick" with "squash" or "s" on the second commit. 
save and edit commit message to something that makes sense ("Updated Italian translation" for example)
Force push to update your remote branch. 
`push -f`

File: translations/monero-core_it.ts [hunk 1]
```diff
@@ -192,7 +192,7 @@
     <message>
         <location filename="../pages/Dashboard.qml" line="102"/>
         <source>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;                        looking for security level and address book? go to &lt;a href=&apos;#&apos;&gt;Transfer&lt;/a&gt; tab</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;                        Cerchi il livello di sicurezza e la lista degli indirizzi? vai qui: &lt;a href=&apos;#&apos;&gt;Invia&lt;/a&gt; tab</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: `type="unfinished"` should be removed.

File: translations/monero-core_it.ts [hunk 2]
```diff
@@ -519,107 +519,87 @@
...
         <location filename="../pages/Mining.qml" line="153"/>
         <source>Start mining</source>
-        <translation type="unfinished">Inizia mining</translation>
+        <translation>Inizia a minare</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="163"/>
         <source>Error starting mining</source>
-        <translation type="unfinished">Errore in partenza mining</translation>
+        <translation type="unfinished">Errore durante inizio mining</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 3]
```diff
@@ -519,107 +519,87 @@
...
-        <translation type="obsolete">Minare ti da anche una piccola possibilità di guadagnare Monero.&lt;br&gt;</translation>
-    </message>
-    <message>
-        <source>Your computer will search for Monero block solutions.&lt;br&gt;</source>
-        <translation type="obsolete">Il tuo PC cercherà di risolvere un blocco di Monero.&lt;br&gt;</translation>
-    </message>
-    <message>
-        <source>If you find a block, you will get the associated reward.&lt;br&gt;</source>
-        <translation type="obsolete">Se trovi un blocco riceverai  la ricompensa associata </translation>
+        <translation>(disponibile solo per daemons locali</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Missing `)` at the end.

File: translations/monero-core_it.ts [hunk 4]
```diff
@@ -652,7 +632,7 @@
     <message>
         <location filename="../components/NetworkStatusItem.qml" line="60"/>
         <source>Invalid connection status</source>
-        <translation type="unfinished">Stato connessione invalido</translation>
+        <translation type="unfinished">Connessione invalida</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 5]
```diff
@@ -834,17 +810,17 @@
     <message>
         <location filename="../pages/Receive.qml" line="282"/>
         <source>Generate payment ID for integrated address</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Genera ID Pagamento per l'indirizzo integrato</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 6]
```diff
@@ -854,12 +830,12 @@
...
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;Questo è un semplice tracker per le vendite:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Clicca "Genera" per creare un ID Pagamento casuale per un nuovo cliente&lt;/p&gt; &lt;p&gt;Fai scannerizzare al cliente il codice QR per effettuare un pagamento (Se il cliente possiede un QR scanner).&lt;/p&gt;&lt;p&gt;Questa pagina scannerizzerà automaticamente la blockchain e il txpool in cerca di transazioni in entrata usando il codice QR. Se inserisci una qauntità controllerà anche se ci sono transazioni in entrata per l'ammontare specificato.&lt;/p&gt;It&apos;Sta a te decidere se accettare transazioni non confermate.</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 7]
```diff
@@ -964,82 +940,82 @@
...
         <location filename="../pages/Settings.qml" line="372"/>
         <source>Version</source>
-        <translation type="unfinished">Versione</translation>
+        <translation>Versione</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="388"/>
         <source>GUI version: </source>
-        <translation type="unfinished">Versione interfaccia grafica:</translation>
+        <translation>Versione interfaccia grafica(GUI):</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please add space before `(GUI)`

File: translations/monero-core_it.ts [hunk 8]
```diff
@@ -1227,45 +1203,45 @@
...
         <location filename="../components/TableDropdown.qml" line="183"/>
         <source>&lt;b&gt;Send to same destination&lt;/b&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Invia allo stesso indirizzo&lt;/b&gt;</translation>
     </message>
     <message>
         <location filename="../components/TableDropdown.qml" line="184"/>
         <source>&lt;b&gt;Find similar transactions&lt;/b&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">&lt;b&gt;Trova transazioni simili&lt;/b&gt;</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 9]
```diff
@@ -1821,44 +1793,44 @@ Ringsize: </source>
...
         <location filename="../wizard/WizardFinish.qml" line="60"/>
         <source>Backup seed</source>
-        <translation type="unfinished">Seed di backup</translation>
+        <translation>Seed di backup</translation>
     </message>
     <message>
         <location filename="../wizard/WizardFinish.qml" line="61"/>
         <source>Wallet path</source>
-        <translation type="unfinished">Cartella del portafoglio</translation>
+        <translation type="unfinished">Percorso del portafoglio</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 10]
```diff
@@ -2287,27 +2255,27 @@ Ringsize: </source>
...
         <location filename="../main.qml" line="1287"/>
         <source>Stop daemon</source>
-        <translation type="unfinished">Arresta daemon</translation>
+        <translation>Arresta daemon</translation>
     </message>
     <message>
         <location filename="../main.qml" line="1322"/>
         <source>New version of monero-wallet-gui is available: %1&lt;br&gt;%2</source>
-        <translation type="unfinished">Una nuova versione interfaccia grafica disponibile: %1&lt;br&gt;%2</translation>
+        <translation type="unfinished">Una nuova versione dell'interfaccia grafica è disponibile: %1&lt;br&gt;%2</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Please remove `type="unfinished"`

File: translations/monero-core_it.ts [hunk 11]
```diff
@@ -2287,27 +2255,27 @@ Ringsize: </source>
     <message>
         <location filename="../main.qml" line="719"/>
         <source>This address received %1 monero, with %2 confirmation(s).</source>
-        <translation type="unfinished">Questo indirizzo ha ricevuto %1 monero con %2 conferme.</translation>
+        <translation>Questo indirizzo ha ricevuto %1 monero con %2 conferme.</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: Does "conferme" work for both plural and singular?

File: translations/monero-core_it.ts [hunk 12]
```diff
@@ -16,17 +16,17 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="63"/>
         <source>&lt;b&gt;Tip tekst test&lt;/b&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Tip tekst test&lt;/b&gt;</translation>
```
Comment (jonathancross) [inline translations/monero-core_it.ts]: This should not be "translated" as it is not real text.
Will be removed in the future and marking it as finished will make that harder.

**Comment by erciccione** [comment]
@jonathancross @Jaqueeee Thanks for the help guys, i'll fix as soon as i can

**Comment by erciccione** [comment]
@Jaqueeee i'm stuck. I don't understand why when i run    
```

**Comment by erciccione** [comment]
@Jaqueeee  I'm stuck, i don't understand why when i run
```
git rebase -i HEAD~4
```
i see the list of all the commits, and not even all mine:
```
pick 349f3fd test
pick 6a0ed3c Change wallet languages to native names and scripts
pick 6beadba Activate German in languages.xml
pick af1edc0 fix version info
pick 5ec575b Do not output seed in log
pick a046163 made changes so that start-gui.sh script can be called from anywhe$
pick de4ff29 Capitalize Create-tx-file translations
pick cfa2f81 Fix Italian translation
pick 72a7f13 Small edits to Build Instructions
pick 127790e set correct seed lang when recovering from keys
pick 41b2c28 Fallback to english seed lang if not set
pick a3e0589 Updates and fixes to the Dutch translation
pick e132ba0 updates (3) italian translation
pick 1375305 UX: remove wide header
pick 443eb47 UX: move 'rescan spent' to settings and rename to 'Rescan wallet b$
pick 9037eed DaemonManager: support dataDir parameter + validator
pick 62e1dc4 Setttings: blockchain folder location
pick 08b54f6 Restore wallet: Adding space after 'Your wallet is stored in'
pick 8e986b3 German translation fixes (from PR #724)
```
what am i doing wrong? is it better if i delete this and make a new PR?

**Comment by erciccione** [comment]
became a mess, i close this and i'm making a new PR

**Pull request closed without merging**